### PR TITLE
feat: Move Debug Panel outside the main content container

### PR DESCRIPTION
**Pull request by srinaath**
## Description

Move Debug Pane at the same level of the project tree.

CC @mewa1024 

## Task Item

Refs #6749

## Screenshots

### CODE REVIEW

File: Composer/packages/client/src/pages/design/DesignPage.tsx [hunk 1]
```diff
@@ -35,37 +51,39 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
...
-                <VisualPanel projectId={activeBot} />
-                <PropertyPanel isSkill={activeBot !== projectId} projectId={activeBot} />
-              </Split>
-            </div>
-          </Conversation>
-          <DebugPanel />
-        </div>
-      </Split>
-      <Modals projectId={activeBot} />
+    <div css={root} data-testid={props['data-testid']}>
```
Comment (hatpick) [inline Composer/packages/client/src/pages/design/DesignPage.tsx:54]: why does this need to come from props?

File: Composer/packages/client/src/pages/design/DebugPanel/DebugPanel.tsx [hunk 2]
```diff
@@ -124,67 +129,69 @@ export const DebugPanel: React.FC = () => {
...
+            role="button"
+            tabIndex={0}
+            onClick={onDebugPaneClick}
+            onKeyPress={onDebugPaneClick}
           />
+          <div css={rightBarStyle} data-testid="header__right">
+            <IconButton
+              iconProps={{ iconName: isPanelExpanded ? 'ChevronDown' : 'ChevronUp' }}
+              styles={{ root: { height: '100%' } }}
+              title={formatMessage('Collapse debug panel')}
```
Comment (hatpick) [inline Composer/packages/client/src/pages/design/DebugPanel/DebugPanel.tsx:177]: shouldn't this dynamically change based on ```isPanelExpanded```?

File: Composer/packages/client/src/components/Page.tsx [hunk 3]
```diff
@@ -64,7 +64,7 @@ export const headerContent = css`
 
 export const main = (hasRenderHeaderContent) => css`
   margin-left: 2px;
-  height: ${hasRenderHeaderContent ? 'calc(100vh - 181px)' : 'calc(100vh - 165px)'};
+  max-height: ${hasRenderHeaderContent ? 'calc(100vh - 181px)' : 'calc(100vh - 165px)'};
```
Comment (hatpick) [inline Composer/packages/client/src/components/Page.tsx:67]: I have issues with 181px, where is it coming from? 😄 , why not 180px?

**Comment by hatpick** [comment]
+ @srinaath videos attached to your PRs are pretty small, I dont know about others but I appreciate a larger size video. ty

File: Composer/packages/client/src/components/Page.tsx (hunk 3 above)
Comment (srinaath) [reply Composer/packages/client/src/components/Page.tsx:67]: Yeah. I dont actually know. It's a "magic" number. The changes I made actually dont affect those. I can add a TODO to look into this.

File: Composer/packages/client/src/pages/design/DesignPage.tsx (hunk 1 above)
Comment (srinaath) [reply Composer/packages/client/src/pages/design/DesignPage.tsx]: It doesn't have to at all. It's a remnant from code copy.

File: Composer/packages/client/src/components/Page.tsx (hunk 3 above)
Comment (srinaath) [reply Composer/packages/client/src/components/Page.tsx:68]: [Look into magic numbers in styles · Issue #6873 · microsoft/BotFramework-Composer (github.com)](https://github.com/microsoft/BotFramework-Composer/issues/6873)

**Review by hatpick — APPROVED** [review]

**Pull request merged**